bluld-1.1.1 fails to build

openwrt

Description:
```
make[6]: Entering directory '/home/ByteEnable/temp/openwrt/build_dir/target-arm_cortex-a9+neon_musl_eabi/bluld-1.1.1'
[ 11%] Building CXX object CMakeFiles/bluld.dir/blinkstick-userspace-led-daemon/src/LEDBinding.cpp.o
/home/ByteEnable/temp/openwrt/build_dir/target-arm_cortex-a9+neon_musl_eabi/bluld-1.1.1/blinkstick-userspace-led-daemon/src/LEDBinding.cpp: In member function 'void BlinkstickUserspace::LEDBinding::registerUserSpaceLED()':
/home/ByteEnable/temp/openwrt/build_dir/target-arm_cortex-a9+neon_musl_eabi/bluld-1.1.1/blinkstick-userspace-led-daemon/src/LEDBinding.cpp:57:10: error: 'char* strncpy(char*, const char*, size_t)' specified bound 64 equals destination size [-Werror=stringop-truncation]
   strncpy(dev.name, mName.c_str(), LED_MAX_NAME_SIZE);
   ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
make[6]: *** [CMakeFiles/bluld.dir/build.make:95: CMakeFiles/bluld.dir/blinkstick-userspace-led-daemon/src/LEDBinding.cpp.o] Error 1
make[6]: Leaving directory '/home/ByteEnable/temp/openwrt/build_dir/target-arm_cortex-a9+neon_musl_eabi/bluld-1.1.1'
make[5]: *** [CMakeFiles/Makefile2:95: CMakeFiles/bluld.dir/all] Error 2
make[5]: Leaving directory '/home/ByteEnable/temp/openwrt/build_dir/target-arm_cortex-a9+neon_musl_eabi/bluld-1.1.1'
make[4]: *** [Makefile:149: all] Error 2
make[4]: Leaving directory '/home/ByteEnable/temp/openwrt/build_dir/target-arm_cortex-a9+neon_musl_eabi/bluld-1.1.1'
make[3]: *** [Makefile:50: /home/ByteEnable/temp/openwrt/build_dir/target-arm_cortex-a9+neon_musl_eabi/bluld-1.1.1/.built] Error 2
make[3]: Leaving directory '/home/ByteEnable/temp/openwrt/feeds/packages/utils/bluld'
time: package/feeds/packages/bluld/compile#0.41#0.07#0.47
    ERROR: package/feeds/packages/bluld failed to build.
make[2]: *** [package/Makefile:116: package/feeds/packages/bluld/compile] Error 1
make[2]: Leaving directory '/home/ByteEnable/temp/openwrt'
make[1]: *** [package/Makefile:110: /home/ByteEnable/temp/openwrt/staging_dir/target-arm_cortex-a9+neon_musl_eabi/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/ByteEnable/temp/openwrt'
make: *** [/home/ByteEnable/temp/openwrt/include/toplevel.mk:230: world] Error 2
```

--- a/utils/bluld/Makefile
+++ b/utils/bluld/Makefile
@@ -1,18 +1,19 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=bluld
-PKG_VERSION:=1.1.1
+PKG_VERSION:=1.1.2
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://github.com/ktgeek/$(PKG_NAME)/releases/download/v$(PKG_VERSION)
-PKG_HASH:=995474bca662cfd8a693a3ab48bdddb2e09c258183a9c29aeddfa341a5082d0e
+PKG_HASH:=94e42999bf701ea01296d614fe298af3bd5b91d3bdab6adda7ecc24af69f8230
 
 PKG_MAINTAINER:=Keith T. Garner <kgarner@kgarner.com>
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE
 
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 EXTRA_CXXFLAGS = -I$(LINUX_DIR)/include

